server/lib/logging/errbit-transport.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+const Errbit = require('./errbit')
+const createAbstractTransport = require('pino-abstract-transport')
+
+module.exports = ({ severity, enabled, host, projectId, projectKey, environment, version }) => {
+  const errbit = new Errbit({ severity, enabled, host, projectId, projectKey, environment, version })
+
+  return createAbstractTransport(async source => {
+    for await (const data of source) {
```

Comment:
should this be a `Promise.all`?

Reply:
This is doing [async iteration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of) (was new to me). I don't think Promise.all will work here as each promise comes in from the source (a Duplex stream) at different times.